### PR TITLE
SoapWrapper should try unwrapping with soap version 1.1 too (#6032)

### PR DESCRIPTION
**Pull request by jkosternl**
* Prevents bug of reusing stored SoapVersion from previous SoapWrappers

### CODE REVIEW

File: core/src/test/java/org/frankframework/pipes/SoapWrapperPipeTest.java [hunk 1]
```diff
@@ -516,13 +504,80 @@ void shouldUseConfiguredPipelineSoapVersionAuto() throws Exception {
...
+		// Assert: but the output is without SOAP namespaces, as it is unwrapped.
+		String result = pipeRunResult3.getResult().asString();
+		assertNotNull(result);
+		assertFalse(result.contains(SoapVersion.SOAP11.namespace), "result should be unwrapped for sure");
+		assertFalse(result.contains(SoapVersion.SOAP12.namespace), "result should be unwrapped for sure");
+		assertTrue(result.contains("FindDocuments_Response"), "result should contain original message, without namespaces");
+		outputWrapper.stop();
+	}
+
+	private P createWrapperSoapPipe(final SoapVersion soapVersion, boolean started) throws ConfigurationException, PipeStartException {
```
Comment (Yale96) [inline core/src/test/java/org/frankframework/pipes/SoapWrapperPipeTest.java:557]: I only see started being set to true so is it really needed to add a specific param for this?
What happens if it's not started?

File: core/src/test/java/org/frankframework/pipes/SoapWrapperPipeTest.java [hunk 2]
```diff
@@ -516,13 +504,80 @@ void shouldUseConfiguredPipelineSoapVersionAuto() throws Exception {
...
+		wrapPipeSoap.setSoapVersion(soapVersion);
+		pipeline.addPipe(wrapPipeSoap);
+		if (started) {
+			wrapPipeSoap.configure();
+			wrapPipeSoap.start();
+		}
+		return wrapPipeSoap;
+	}
+
+	private void setupUnwrapSoapPipe(final SoapVersion soapVersion, boolean started) throws ConfigurationException, PipeStartException {
```
Comment (Yale96) [inline core/src/test/java/org/frankframework/pipes/SoapWrapperPipeTest.java:572]: The same counts for this

File: core/src/test/java/org/frankframework/pipes/SoapWrapperPipeTest.java (hunk 1 above)
Comment (jkosternl) [reply core/src/test/java/org/frankframework/pipes/SoapWrapperPipeTest.java:557]: It is set to false in my new test: `shouldUnwrapEvenThoughSessionHasDifferentSoapVersionStored`
It is postponed in my test, so I can configure more variables/settings and then run the configure&start inside my unit test. Changing settings after a configure is not a good idea (and mostly not taken into account).

File: core/src/test/java/org/frankframework/pipes/SoapWrapperPipeTest.java (hunk 2 above)
Comment (jkosternl) [reply core/src/test/java/org/frankframework/pipes/SoapWrapperPipeTest.java:572]: It is set to false in my new test: `shouldUnwrapEvenThoughSessionHasDifferentSoapVersionStored`
It is postponed in my test, so I can configure more variables/settings and then run the configure&start inside my unit test. Changing settings after a configure is not a good idea (and mostly not taken into account).

**Review by Yale96 — APPROVED** [review]

File: core/src/main/java/org/frankframework/soap/SoapWrapper.java [hunk 3]
```diff
@@ -168,6 +167,10 @@ private String extractMessageWithTransformers(TransformerPool transformerS11, Tr
 		String extractedMessage;
 		if (soapVersion == SoapVersion.SOAP12) {
 			extractedMessage = transformerS12.transform(message.asSource());
+			// If session had the wrong SOAP version stored (e.g. multiple SoapWrappers), try SOAP 1.1 too.
```
Comment (tnleeuw) [inline core/src/main/java/org/frankframework/soap/SoapWrapper.java:170]: This works for now but for future I think we should find a better solution.
Can you please add a TODO comment for that in the code, at least in master?
(we don't need such TODO in the bugfix branches).

**Review by tnleeuw — APPROVED** [review]

**Review by tnleeuw — APPROVED** [review]

**Pull request merged**